CharityPortal/api/handlers/home_handler.go

```diff
@@ -0,0 +1,16 @@
+package handlers
+
+import (
+	"charity_portal/web/views"
+	"net/http"
+)
+
+type HomeHandler struct{}
+
+func NewHomeHandler() *HomeHandler {
+	return &HomeHandler{}
+}
+
+func (ph *HomeHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	views.Base().Render(r.Context(), w)
+}
```

Comment:
TBH I don't get this. Is this how Templ works? You pass empty struct to mux as handler, then ServeHTTP takes over handling the response?